src/components/pages/devices/flyouts/deviceNew/deviceNew.js

```diff
@@ -401,125 +401,132 @@ export class DeviceNew extends LinkedComponent {
+      <Flyout header={t('devices.flyouts.new.title')} t={t} onClose={() => this.onFlyoutClose('Devices_TopXCloseClick')}>
+        <Protected permission={permissions.createDevices}>
+          <form className="devices-new-container" onSubmit={this.apply}>
+            <div className="devices-new-content">
+              <FormGroup>
+                <FormLabel>{t(deviceOptions.labelName)}</FormLabel>
+                <Radio link={this.deviceLink} value={deviceOptions.device.value} onChange={this.deviceChange}>
+                  {t(deviceOptions.device.labelName)}
+                </Radio>
+                <Radio link={this.deviceLink} value={deviceOptions.edgeDevice.value} onChange={this.deviceChange}>
+                  {t(deviceOptions.edgeDevice.labelName)}
+                </Radio>
+              </FormGroup>
+              {
+                isEdgeDevice &&
                 <FormGroup>
-                  <FormLabel>{t(deviceOptions.labelName)}</FormLabel>
-                  <Radio link={this.deviceLink} value={deviceOptions.device.value} onChange={this.deviceChange}>
-                    {t(deviceOptions.device.labelName)}
+                  <FormLabel>{t(deviceTypeOptions.labelName)}</FormLabel>
+                  <Radio link={this.deviceTypeLink} value={deviceTypeOptions.simulated.value} onChange={this.deviceTypeChange}>
+                    {t(deviceTypeOptions.simulated.labelName)}
                   </Radio>
-                  <Radio link={this.deviceLink} value={deviceOptions.edgeDevice.value} onChange={this.deviceChange}>
-                    {t(deviceOptions.edgeDevice.labelName)}
+                  <Radio link={this.deviceTypeLink} value={deviceTypeOptions.physical.value} onChange={this.deviceTypeChange}>
+                    {t(deviceTypeOptions.physical.labelName)}
                   </Radio>
                 </FormGroup>
-                {
-                  isEdgeDevice &&
+              }
+              {
+                isSimulatedDevice &&
+                <ComponentArray>
+                  <FormGroup>
+                    <FormLabel>{t('devices.flyouts.new.count.label')}</FormLabel>
+                    <FormControl link={this.countLink} type="text" onChange={this.formControlChange} />
+                  </FormGroup>
                   <FormGroup>
-                    <FormLabel>{t(deviceTypeOptions.labelName)}</FormLabel>
-                    <Radio link={this.deviceTypeLink} value={deviceTypeOptions.simulated.value} onChange={this.deviceTypeChange}>
-                      {t(deviceTypeOptions.simulated.labelName)}
+                    <FormLabel>{t('devices.flyouts.new.deviceIdExample.label')}</FormLabel>
+                    <div className="device-id-example">{t('devices.flyouts.new.deviceIdExample.format', { deviceName })}</div>
+                  </FormGroup>
+                  <FormGroup>
+                    <FormLabel id="deviceModelLabel">{t('devices.flyouts.new.deviceModel.label')}</FormLabel>
```

Comment:
Double check but I think FormGroup will add an id for you.

Reply:
done